File: Java/PrimeNumber.java
import java.util.Scanner;

class Program
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter number to check whether it is prime or not");

        int n = scanner.nextInt();
        int a = 0;

        for (int i = 1; i <= n; i++) 
         {
            if (n % i == 0) 
            {
               a++;
            }
         }
         if (a == 2) 
         {
            System.out.println(n + " is prime number");
         } 
         else 
         {
            System.out.println(n + " is not a prime number");
         }
         
         scanner.close();
      }

    }

File: C#/PrimeNumber.cs
//Write a c# program to check prime number.

using System;
   class PrimeNumber 
   {
      public static void Main() 
      {
         int n = 5, a = 0;
         for (int i = 1; i <= n; i++) 
         {
            if (n % i == 0) 
            {
               a++;
            }
         }
         if (a == 2) 
         {
            Console.WriteLine("{0} is a Prime Number", n);
         } 
         else 
         {
            Console.WriteLine("{0} Not a Prime Number", n);
         }
         Console.ReadLine();
      }
   }
